src/3d/qgs3dmapscene.cpp

```diff
@@ -1095,6 +1095,20 @@ QgsDoubleRange Qgs3DMapScene::elevationRange() const
       yMin = std::min( yMin, zRange.lower() );
       yMax = std::max( yMax, zRange.upper() );
     }
+    else if ( layer->type() == Qgis::LayerType::Mesh )
```

Comment:
Could we replace this with a switch? (With no default clause!). It's possible newly introduced layer types (Eg tiled scenes) could trivially implement this method, so an explicit compilation warning is preferable here to bring attention to this code.

Reply:
Sure. Done.

Reply:
@nyalldawson This results in a build failure in fedora 38 ci job.